lib/core/config.js

```diff
@@ -187,6 +187,9 @@ Config.prototype.loadBlockchainConfigFile = function() {
   if (!configFilePath) {
     this.blockchainConfig.default = true;
   }
+  if (!this.blockchainConfig.account) {
```

Comment:
Couldn't this confuse users who try to use `testnet` or something else but then, there is no `accounts` so it boots dev. wouldn't an error make more sense (error with good description)

Reply:
Agree, we shouldn't assume isDev is there is no account, in fact this will be case for anyone using infura for e.g

Reply:
In that case there is already this error:
```Geth error: Unlocking account 95cd9d4af4d472712241cd19b7459c0253a0ad12 | Attempt 1/3
!! Unsupported terminal, password will be echoed.
Passphrase: ```

The one:
TypeError: Cannot read property 'toString' of undefined
    at web3.eth.net.getId.then.id (/home/anthony/code/embark-framework/embark/lib/modules/blockchain_connector/index.js:85:45)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:118:7)

Is fixed by doing `if (networkId`
Do you think that more should be added?

Reply:
Yeah, the first error is really unclear. If there are not accounts and it is not `isDev`, I think we should error with something like: "Account settings are needed for this chain. Please put a valid address and possibly a password in your blockchain config"
Might be good to link the docs too (if we have some for it?)

Reply:
I added the warning